libs/api/metadata-converter/src/lib/iso19139/write-parts.ts

```diff
@@ -267,6 +267,24 @@ export function getMaintenanceFrequencyCode(
       return 'continual'
     case 'periodic':
       return 'periodic'
+    case 'daily':
+      return 'daily'
+    case 'weekly':
+      return 'weekly'
+    case 'fortnightly':
+      return 'fortnightly'
+    case 'monthly':
+      return 'monthly'
+    case 'quarterly':
+      return 'quarterly'
+    case 'biannually':
+      return 'biannually'
+    case 'annually':
+      return 'annually'
+    case 'semimonthly':
+      return 'semimonthly'
+    case 'biennially':
```

Comment:
maybe i am missing something but i don't really understand the purpose of this function, is it only to output the frequency as a string? + it is missing the default case returning null.

Reply:
Yes it seems useless now, maybe because you changed the way frequency code is handled. Dunno

Reply:
I didn't change anything regarding the frequency code handling, i have just allowed 2 more options to be displayed in the dropdown list.